buster/completers/base.py

```diff
@@ -228,7 +228,7 @@ class DocumentAnswerer:
 
     def __init__(
         self,
-        documents_formatter: DocumentsFormatter,
+        documents_formatter: DocumentsFormatterHTML,
```

Comment:
Why? Any class deriving from `DocumentsFormatter` should be fine no?

Reply:
yes, fixed